qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/3-upgrade-mgr-staggered.yaml

```diff
@@ -0,0 +1,18 @@
+teuthology:
+  premerge: |
+            if not yaml.teuthology.variables.fail_fs then reject() end
+upgrade-tasks:
+  sequential:
+  - cephadm.shell:
```

Comment:
```suggestion
    - cephadm.shell:
```

and indent everything below

Reply:
also, 3-upgrade-with-workload.yaml needs its indentation fixed as well (apparently the yaml parser doesn't view this as a problem; we should fix it anyway).

Reply:
Done, PTAL